fairlearn/metrics/_metric_frame.py

```diff
@@ -226,6 +338,9 @@ def overall(self) -> Union[Any, pd.Series, pd.DataFrame]:
             interface when calling programatically, while also reducing
             typing for those using Fairlearn interactively.
         """
+        if self._overall is None:
```

Comment:
Does this mean that once a user has invoked one of these summary functions, they can no longer keep streaming data into the object?

Reply:
We can still keep it, but we will wipe `_overall` and `by_group` so the next call will recompute it.

Reply:
Ahh yes, I see above.